client/src/components/overview/ClickableStar.jsx

```diff
@@ -0,0 +1,9 @@
+export default function(props) {
+  const stars = {
+    clicked: '⭐',
```

Comment:
is this star just an image? Does the following change in package.json "svg-url-loader": "^8.0.0", works for it?

Reply:
This is technically a text! The change in the package.json is for loading the svg for the reviews star, this ClickableStar component is actually something different, it's for allowing the customer to add the product to their favorites